service/spotify_service.go

```diff
@@ -0,0 +1,164 @@
+/*
+	Service focus on Searching information about Music Artist using Spotify API.
+	DOCs: https://developer.spotify.com/documentation/general/guides/authorization/client-credentials/
+*/
+
+package service
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/ErickRodriguezWize/academy-go-q42021/domain/model"
+	spotiferr "github.com/ErickRodriguezWize/academy-go-q42021/errors"
+)
+
+var (
+	// Environment Variables.
+	/* CONST */ SpotifyEndpoint = GetEnvVariable("SPOTIFY_API_ENDPOINT")
+	/* CONST */ SpotifyRefreshEndpoint = GetEnvVariable("SPOTIFY_REFRESH_ENDPOINT")
+	/* CONST */ SpotifyAuhtorizationToken = GetEnvVariable("SPOTIFY_AUTHORIZATION_TOKEN")
+	/* CONST */ SpotifyRefreshToken = GetEnvVariable("SPOTIFY_REFRESH_TOKEN")
+	/* CONST */ LimitArtist = GetEnvVariable("LIMIT_ARTIST")
+)
```

Comment:
The reason behind  "var" instead of Const, is that Go doesn't let you assign the return value from a func into a const. That's the reason why I use var. 

This been said, I already remove all the "Env variables" to a file called config.yaml. This Env Variables are already defined in the main.go, as you suggested. Then I distributed to all controllers from main.go